guides/source/active_record_callbacks.md

```diff
@@ -538,7 +593,19 @@ irb> @user.save # updating @user
 User was saved to database
 ```
 
+### `after_commit` ordering
```

Comment:
@ghiculescu Could you take a look at this? :bow:

Reply:
Yeah, thats currently true.